types/gzip-size/gzip-size-tests.ts

```diff
@@ -1,4 +1,4 @@
-import gzipSize = require('gzip-size');
+import * as gzipSize from 'gzip-size';
```

Comment:
It seems it shouldn't be changed to ES module from commonjs module, because an ES6 module object cannot be invoked as a function or with new(https://stackoverflow.com/questions/39415661/what-does-resolves-to-a-non-module-entity-and-cannot-be-imported-using-this)